code/modules/mob/living/simple_animal/hostile/human/grey.dm

```diff
@@ -20,18 +33,75 @@
 	max_co2 = 0
 	min_n2 = 0
 	max_n2 = 0
+	minbodytemp = 0
 
 	corpse = /obj/effect/landmark/corpse/grey/space
 
-/obj/effect/landmark/corpse/grey
-	name = "grey"
-	mutantrace = "Grey"
-
-/obj/effect/landmark/corpse/grey/space
-	corpsemask = /obj/item/clothing/mask/breath
-	corpsegloves = /obj/item/clothing/gloves/grey
-	corpsesuit = /obj/item/clothing/suit/space/grey
-	corpseuniform = /obj/item/clothing/under/color/grey
-	corpseshoes = /obj/item/clothing/shoes/black
-	corpseback = /obj/item/weapon/tank/oxygen/red
-	corpsehelmet = /obj/item/clothing/head/helmet/space/grey
+	faction = "mothership"
+
+/mob/living/simple_animal/hostile/humanoid/grey/space/Process_Spacemove(var/check_drift = 0)
+	return 1
+
+/mob/living/simple_animal/hostile/humanoid/grey/space/melee/toolbox
+	name = "Grey Explorer"
+	desc = "A thin alien humanoid in a space suit. This one seems to be hostile."
+
+	icon_state = "greyspace_toolbox"
+	icon_living = "greyspace_toolbox"
+	melee_damage_lower = 12
+	melee_damage_upper = 15
+
+	items_to_drop = list(/obj/item/weapon/storage/toolbox/mechanical)
+
+	speak = list("Speak softly, and carry a robust toolbox.","Where did I put that soldering iron?","Mothership guide us...")
+	speak_chance = 5
+
+/mob/living/simple_animal/hostile/humanoid/grey/space/melee/toolbox/Aggro()
+	..()
+	say(pick("I won't allow you to damage mothership equipment!","Time to apply percussive maintenance!"))
+
+	attacktext = "batters"
+	attack_sound = 'sound/weapons/toolbox.ogg'
+
+/mob/living/simple_animal/hostile/humanoid/grey/space/melee/scalpel
+	name = "Grey Explorer"
+	desc = "A thin alien humanoid in a space suit. This one seems to be hostile."
+
+	icon_state = "greyspace_scalpel"
+	icon_living = "greyspace_scalpel"
+	melee_damage_lower = 15
+	melee_damage_upper = 18
+
+	items_to_drop = list(/obj/item/tool/scalpel)
+
+	speak = list("I need a new specimen to dissect.","A dissection is all I need... just one more dissection.","Praise the mothership.")
+	speak_chance = 5
+
+/mob/living/simple_animal/hostile/humanoid/grey/space/melee/scalpel/Aggro()
+	..()
+	say(pick("I need your organs for testing!","You'll make a fine specimen for an operation!"))
```

Comment:
if you want to make these guys speak quacks, you can just add this after your pick:

`, get_language(LANGUAGE_GREY))`

Reply:
Tried it, and it seems to throw back an error saying that get_language is an undefined proc.
![Capture](https://user-images.githubusercontent.com/81007356/146842545-d38a83f7-44ab-41fb-a4e7-f45e3f18e24c.PNG)

Reply:
I'm guessing that means I have to add a line to the page somewhere that defines said proc. Uncertain how to go about that, though.

Reply:
no need, I forgot that proc isn't universal

Instead do this:
`, all_languages[LANGUAGE_GREY]))`

all_languages is a global list so it should be good
I also suggest you add this to the `/humanoid/grey/New()` --
```
languages += all_languages[LANGUAGE_GREY]
```

This should ensure that if someone is bussed into one of these mobs, they will understand grey quacks

Reply:
Okay, I made an attempt to apply the changes in a new commit. Let me know if they look alright or still need to be refined. Tested them and it seemed to work.